integration/http/template

```diff
@@ -0,0 +1,10 @@
+{
+	attributes: [ {{- $first_1 := true -}}
```

Comment:
Would treat this separately to this PR, but the 'first' vars here are a way to add a comma between events but not to the last one.

I hate it, it's clunky and makes the template hard to read. Might we be able to use a function to deal with this?

It's a small QOL thing, but this plays a part in a complex/confusing system - so if it's simple to support something that makes things more readable  then there's value in it IMO.

Reply:
I agree, it's ugly!

I don't think there is any neat way to deal with it (at least haven't found it yet). What do you think about this version though:

```
{
  attributes: [{{range $i, $data := .}}{{if $i}},{{end}}{{.attribute_data}}{{end}}],
  events: [{{range $i, $data := .}}{{if $i}},{{end}}{{prettyPrint .event_data}}{{end}}]
}
```

So iterate with index, for 0 simply don't add comma.

Reply:
That is better!